packages/css-jss/src/index.d.ts

```diff
@@ -0,0 +1,15 @@
+import {Jss, Style} from 'jss'
```

Comment:
```suggestion
import {Jss, JssStyle} from 'jss'
```

I think this is named `JssStyle` now

Reply:
Rebased to master branch and updated imports